app/views/application/index.html.erb

```diff
@@ -1,65 +1,46 @@
-              </div>
-            </div>
-            <br>
-            <%= content_tag :div, class: "j-location_info", data: {loc: location} do %><% end %>
-          <% end %>
-        </div>
-      </div>
+<div class="header">
+  <div class="title">
+    <%= image_tag("sfbbo-logo.png", class: "logo") %>
+    <div class="name">BAY TRAIL BIRD MAPPING</div>
+  </div>
+  <div class="nav">
+    <div class="nav-btn"><i class="fa-solid fa-magnifying-glass"></i></div>
+    <div class="nav-btn" id="navigation">
+      <i class="fa-solid fa-ellipsis-vertical"></i>
+      <nav>
+        <%= link_to "Donate", "/donation" %>
+        <%= link_to "Locations", "/locations" %>
+        <%= link_to "Tips", "/tips" %>
+      </nav>
     </div>
-
-    <div class="col-md-8">
-        <div id="mapid" style="height:500px"></div>
+    <div class="nav-btn info"><i class="fa-solid fa-info"></i></div>
+  </div>
+</div>
+<main>
+  <div id="leaflet-map"></div>
+</main>
+<div class="overlay" hidden>
+  <div class="expand-icon"></div>
+  <div class="content">
+    <div class="bird-name">Clark’s Grebe</div>
+    <div class="science-name">Aechmophorus clarkii</div>
+    <div class="bird-img-wrapper">
+      <img class="bird-img" src="/duck.png">
     </div>
-
-    <div class="col-md-2">
-      <div id="js-com_bird_info" class="card">
-      </div>
+    <div class="location-wrapper">
+      <div class="location">Loc</div>
+      <div class="separator"><i class="fa-solid fa-circle"></i></div>
+      <div class="distance">5 mi</div>
     </div>
-  </div>
-
-  <div class="row">
-    <div class="col-md-2"></div>
-    <div class="col-md-8">
-      <div id="RecentBirdSightings">
-          <h3 class="text-center">Recent Bird Sightings</h3>
-        <div id="RecentBirdList">
-          <div id="header" class="text-center">
-            Click on a location pin to see recent bird sightings there in the past 30 days.
-          </div>
-          <div class="scrolling-wrapper">
-            <div class="card"></div>
-            <div class="card"></div>
-            <div class="card"></div>
-            <div class="card"></div>
-            <div class="card"></div>
-            <div class="card"></div>
-            <div class="card"></div>
-            <div class="card"></div>
-            <div class="card"></div>
-          </div>
-        </div>
+    <div class="description-wrapper">
+      <div class="description-header">Description:</div>
+      <div class="description-content">
+        Clark's grebe is a North American waterbird species in the grebe family. Until the 1980s, it was thought to be a pale morph of the western grebe, which it resembles in size, range, and behavior. Intermediates between the two species are known.
       </div>
     </div>
   </div>
 </div>
-<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js"></script>
-<script src="app/javascript/channels/index.js"></script>
-<%= render "footer" %>
\ No newline at end of file
+
```

Comment:
Is the only change the type=""? Thats not needed in modern HTML, JS is the default.